hbase-server/src/main/java/org/apache/hadoop/hbase/favored/FavoredNodeLoadBalancer.java

```diff
@@ -87,7 +87,8 @@ public synchronized void initialize() throws HBaseIOException {
   }
 
   @Override
-  public List<RegionPlan> balanceCluster(Map<ServerName, List<RegionInfo>> clusterState)  {
+  public List<RegionPlan>
+      balanceCluster(Map<TableName, Map<ServerName, List<RegionInfo>>> clusterLoadPerTable) {
```

Comment:
The balancer should only need to override balanceTable method?

Reply:
yes , but method 'balanceCluster' of this class not invoke 'balanceTable' , so just override balanceTable with null return

Reply:
Didn't get your point. Please add javadoc comment about why retrun null here.